src/layers/fp64/extruded-choropleth-layer/extruded-choropleth-layer.js

```diff
@@ -0,0 +1,449 @@
+// Copyright (c) 2015 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+import {Layer, assembleShaders} from '../../../lib';
+import {GL, Model, Program, Geometry} from 'luma.gl';
+import {fp64ify} from '../../../lib/utils/fp64';
+
+const glslify = require('glslify');
+
+import {vec3} from 'gl-matrix';
+
+import earcut from 'earcut';
+import flattenDeep from 'lodash.flattendeep';
+
+const ATTRIBUTES = {
```

Comment:
We've removed the ATTRIBUTES from the other layer and places these directly in attributeManager.add()